java/pypowsybl/src/main/java/com/powsybl/python/rao/RaoCFunctions.java

```diff
@@ -127,8 +125,11 @@ public static PyPowsyblApiHeader.ArrayPointer<CCharPointer> serializeRaoResultsT
             RaoResult raoResult = ObjectHandles.getGlobal().get(raoResultHandle);
             Crac crac = ObjectHandles.getGlobal().get(cracHandle);
             try (ByteArrayOutputStream output = new ByteArrayOutputStream()) {
-                new RaoResultJsonExporter().exportData(
-                    raoResult, crac, Set.of(Unit.MEGAWATT), output);
+                ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
```

Comment:
The variable outputstream seems to be unused.

Reply:
Removed